.github/workflows/ci.yml

```diff
@@ -130,6 +130,12 @@ jobs:
           CAKE_TEST_AUTOQUOTE=1 vendor/bin/phpunit --testsuite=database
         fi
 
+    - name: Prefer lowest check
+      run: |
+        if ${{ matrix.prefer-lowest == 'prefer-lowest' }}; then
```

Comment:
Couldn't this be an `if` condition for the step instead of inline bash code? See line 110 for an example.

Reply:
Sure, feel free to amend this PR as you see fit. I am not so experienced in this bash scripting.